test/vcr/vcr_cassettes/service_bus/test_0001_should_be_able_to_send_a_message_with_SAS_to_a_queue.yml

```diff
@@ -0,0 +1,169 @@
+---
```

Comment:
Cool. Makes sense. Thanks!

---
In reply to: [76116475](https://github.com/Azure/azure-sdk-for-ruby/pull/425#discussion_r76116475) [](ancestors = 76116475)